Marlin/Configuration.h

```diff
@@ -2027,12 +2027,12 @@
 #endif
 
 #if ENABLED(KNUTWURST_4MAXP2)
-  #define X_MIN_POS -6
+  #define X_MIN_POS -8
   #define Y_MIN_POS 0
   #define Z_MIN_POS 0
   #define X_BED_SIZE 270
-  #define Y_BED_SIZE 210
-  #define Z_MAX_POS 190
+  #define Y_BED_SIZE 205
+  #define Z_MAX_POS 205
```

Comment:
The DirectDrive could crash on the top if is value is applied to the v2.

Reply:
Better I push a new config with separated values for the 4MaxPro1